vite.config.ts

```diff
@@ -4,4 +4,5 @@ import react from '@vitejs/plugin-react-swc'
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
+  base: '/web/',
```

Comment:
Probably `base` is not needed as the frontend will be on the root domain 4rooms.pro/

Reply:
base: '/4rooms.pro/',
and if we tried like this?